### DISCUSSION TITLE
String quotation

### DISCUSSION THREAD

**Discussion by JustusJG** [TodePond/GulfOfMexico#425] [Ideas]
Why am I so limited when it comes to creating strings? Especially with long strings, I often encounter the issue of not knowing whether I used single quotes or double quotes at the beginning by the time I reach the end. Then I have to guess or go back to the beginning of the string to check. This leads to a lot of time wasted and errors during compilation. I know I can omit the quotes, but how am I supposed to know where my string begins and ends then? Therefore, I request that a string that starts with single quotes (') should also be allowed to end with double quotes (") and vice versa.

**Comment by TodePond** [comment]
wow that sounds very revolutionary

dangerous idea
